### DISCUSSION TITLE
Changing the thickness of CircularProgress widget

### DISCUSSION THREAD

**Discussion by SourishBiswas23** [Aylur/ags#638] [Q&A]
Hi. Does anyone know how to change the thickness of the graph in the CircularProgress Widget? I have tried changing the border width, padding, margin, etc but it doesnt seem to work and the docs also dont say anything about it.

**Comment by Aiz0** [comment]
You can change the thickness via the font size

example:
```tsx
 <circularprogress
      value={1}
      css={"font-size:4px"}
    >
```

**Comment by SourishBiswas23** [comment]
Thanks a lot for answering this.
